pkgs/development/python-modules/arxiv/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  feedparser,
+  mock,
+  pytestCheckHook,
+  requests,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "arxiv";
+  version = "2.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "lukasschwab";
+    repo = "arxiv.py";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Niu3N0QTVxucboQx1FQq1757Hjj1VVWeDZn7O7YtjWY=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    feedparser
+    requests
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mock
+  ];
+
+  pythonImportsCheck = [ "arxiv" ];
+
+  meta = rec {
+    description = "Python wrapper for the arXiv API";
+    homepage = "https://github.com/lukasschwab/arxiv.py";
+    changelog = "${homepage}/releases/tag/${version}";
```

```suggestion
  meta = {
    description = "Python wrapper for the arXiv API";
    homepage = "https://github.com/lukasschwab/arxiv.py";
    changelog = "https://github.com/lukasschwab/arxiv.py/releases/tag/${version}";
```

Let's stick to this most simple and common implementation.